spaces/domains.json

```
@@ -4,6 +4,7 @@
   "dao.kuma-inu.com": "kuma-inu.eth",
   "dao.dsla.network": "dsla.eth",
   "vote.goodghosting.com": "goodghosting.eth",
+  "vote.moondao.com": "tomoondao.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.moondao.com": "tomoondao.eth",
```